pkg/sub_controller/feproxy/feproxy_configmap.go

```diff
@@ -113,6 +114,26 @@ http {
     }
 
     location @handle_redirect {
+      if ($upstream_http_location ~ "%[3]s") {
```

Comment:
how about
```
set $pass_body on;
if ( xxx ) {
    set $pass_body off;
}

proxy_pass_request_body $pass_body;
```

Reply:
`if` can be used in rewrite context. See https://nginx.org/en/docs/http/ngx_http_rewrite_module.html

Reply:
just need to check if it is a FE redirect and set $pass_body accordingly, no need to do rewrite.

Reply:
yes, I tried. `if` is not allowed to use without rewrite.